src/main/java/com/tabnine/userSettings/AppSettingsComponent.kt

```diff
@@ -31,6 +32,18 @@ class AppSettingsComponent {
         JBCheckBox("Enable auto-importing packages when selecting Tabnine suggestions", true)
     private val binariesFolderOverrideComponent = JXTextField(StaticConfig.getDefaultBaseDirectory().toString())
     private val cloud2UrlComponent = JBTextField()
+    private val businessDivisionComboBox = ComboBox(
+        arrayOf(
+            "",
```

Comment:
Can we loose this empty option and leave only the default? 
This will help
Indicate interaction